app/mailers/agent_notifications/conversation_notifications_mailer.rb

```diff
@@ -1,21 +1,27 @@
 class AgentNotifications::ConversationNotificationsMailer < ApplicationMailer
-  default from: ENV.fetch('MAILER_SENDER_EMAIL', 'accounts@chatwoot.com')
-  layout 'mailer'
-
   def conversation_creation(conversation, agent)
     return unless smtp_config_set_or_development?
 
     @agent = agent
     @conversation = conversation
     subject = "#{@agent.name}, A new conversation [ID - #{@conversation.display_id}] has been created in #{@conversation.inbox&.name}."
-    mail(to: @agent.email, subject: subject)
+    send_mail_with_liquid(to: @agent.email, subject: subject) and return
   end
 
   def conversation_assignment(conversation, agent)
     return unless smtp_config_set_or_development?
 
     @agent = agent
     @conversation = conversation
-    mail(to: @agent.email, subject: "#{@agent.name}, A new conversation [ID - #{@conversation.display_id}] has been assigned to you.")
+    subject = "#{@agent.name}, A new conversation [ID - #{@conversation.display_id}] has been assigned to you."
```

Comment:
Can/Shall we fetch this from a translation file? Does it make sense?

Reply:
translation or a custom db value makes sense.

Reply:
Will take this up a part of making emails translatable ticket.